### ISSUE TITLE
when there is only a single node consider collapsing nodes/api-node into node

### ISSUE THREAD

**Issue by Jaguar0625** [nemtech/symbol-bootstrap#132]
**Is your feature request related to a problem? Please describe.**
I am always frustrated when bootstrap calls dual node `api-node` and adds extra subdirectory. 

**Describe the solution you'd like**
When there is only single node, collapse `nodes/api-node` path into `node`
[optional]: add separate folder name dual-node (along with api-node and peer-node)

**Comment by rg911** [comment]
Yeah, agree, this is kind of confusing to new users.

**Comment by fboucquez** [comment]
Remember that bootstrap supports multiple nodes, databases, rests, etc. List of one is a special case. 

I could add a if 1 then root level folder case but this is a folder structure change. Some people may be expecting the current folder structure for some of their scripts, although this is not recommended as the structure should be a private implementation.

One example is the nemtus ad-hoc fast sync from this chat

https://nemgroup-workspace.slack.com/archives/C015AEQ9ZL2/p1611567363058900?thread_ts=1611567363.058900&cid=C015AEQ9ZL2
A folder structure change would affect the --upgrade option requiring a full sync.

**Comment by Jaguar0625** [comment]
> List of one is a special case.

Agreed, but I also think it's **common** case, at least initially, so it might be worth adding some special handling to make it a bit more user friendly.

> Some people may be expecting the current folder structure for some of their scripts

Since it is not recommended, I'd not worry too much about it.

> The name comes from the custom preset, it's not hardcoded in the code.

That's good to know. If we have separate dual-node preset, I'd suggest changing the name in it.
